engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -102,7 +100,7 @@ trait Backend {
    * Essentially turns a Call object + CallInputs into a BackendCall
    */
   def bindCall(jobDescriptor: BackendCallJobDescriptor,
```

Comment:
Yeah, I'm not super familiar with the way the abort function registration work so I wasn't sure.
Should we just replace whatever is calling `bindCall` with a `jobDescriptor.copy(abortFunction = abortFunction)` ?